global.pri

```diff
@@ -309,10 +306,10 @@ macx-clang-libc++ {
     QMAKE_OBJECTIVE_CFLAGS += -mmacosx-version-min=$$QMAKE_MACOSX_DEPLOYMENT_TARGET
     QMAKE_OBJECTIVE_CXXFLAGS += -stdlib=libc++ -mmacosx-version-min=$$QMAKE_MACOSX_DEPLOYMENT_TARGET
     c++11 {
```

Comment:
since c++17 is meant to be a requirement, shouldn't we just skip all these conditions and just burn in the c++17 stuff?

Reply:
Good point. Done.